### ISSUE TITLE
Firefox extension mgmt permission errors

### ISSUE THREAD

**Issue by ianb** [mozilla/openwebapps#23]
Installing applications with the Firefox extension, I cannot see them on the dashboard (at https://myapps.mozillalabs.com/) -- running navigator.apps.mgmt.list() seems to return a permission error:

```
["permissionDenied", "to access open web apps management apis, you must be on the same domain as the application repository"]
```

**Comment by anantn** [comment]
Add myapp.mozillalabs.com to management domains. Closed by 134b497dbd60e2d3b1765eed69485c9908cd3322

**Comment by anantn** [comment]
New version of add-on has been pushed to AMO, a manual re-install/upgrade is required.
